roles/lib_openshift/src/test/unit/oc_sdnvalidator.py

```diff
@@ -0,0 +1,481 @@
+#!/usr/bin/env python2
+'''
+ Unit tests for oc sdnvalidator
+'''
+# To run
+# ./oc_sdnvalidator.py
+#
+# ....
+# ----------------------------------------------------------------------
+# Ran 4 tests in 0.002s
+#
+# OK
```

Comment:
Hmm, this comment seem unnecessary to me. How to run tests is described in CONTRIBUTING.md.
However, I'm not opposed to having the `if __name__ ...` at the end if that makes it convenient for you to run the tests.